### DISCUSSION TITLE
I have a problem with the unpack command

### DISCUSSION THREAD

**Discussion by harrypik2** [cfig/Android_boot_image_editor#114] [Q&A]
i use win 10 installed all as required. But I still get an error when I run the unpack command. Is there any solution for this?
![image](https://user-images.githubusercontent.com/117698146/222940044-d81a77c6-45f1-434c-9682-c6d54c1d9ac2.png)
![image](https://user-images.githubusercontent.com/117698146/222940054-bd95e0cf-79fd-4335-8136-f8b8d3722a7c.png)

**Comment by cfig** [comment]
HI @harrypik2 ,
You haven't installed java yet.
If you are using debian/ubuntu variants, try 
```
apt install git device-tree-compiler lz4 xz-utils zlib1g-dev openjdk-17-jdk gcc g++ python3 python-is-python3 p7zip-full
```
Then check your java version with:
`java -version`.

**Reply by harrypik2** [reply]
![image](https://user-images.githubusercontent.com/117698146/224055128-280d1dfe-1777-422d-92da-6a3bbdb70fc2.png)
I do java programming, so there are several versions of java available on my pc.

**Reply by cfig** [reply]
I did testings with JDK11 and JDK17, the most recent 2 LTS versions.
In the first picture, you run the command in WSL?

**Reply by harrypik2** [reply]
idk . It appeared after i followed the instructions for windows.
![image](https://user-images.githubusercontent.com/117698146/224208544-f28dd505-b0c0-4cf8-8e7a-2452b63588c8.png)
